src/CSharp/CodeCracker/Design/MakeMethodStaticAnalyzer.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System.Linq;
+using System.Collections.Immutable;
+
+namespace CodeCracker.CSharp.Design
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class MakeMethodStaticAnalyzer : DiagnosticAnalyzer
+    {
+        internal const string Title = "Use static method";
+        internal const string MessageFormat = "Make '{0}' method static.";
+        internal const string Category = SupportedCategories.Design;
+        const string Description = "If you are not referencing any instance variable your method may become static. That will require";
```

Comment:
It looks like something is missing in `Description` string or am I wrong? :question: